firebase-database/src/androidTest/java/com/google/firebase/database/core/persistence/PersistenceTest.java

```diff
@@ -881,7 +883,8 @@ public void ackForServerValueYieldsResolvedValueInCache() throws Throwable {
     new WriteFuture(ref, ServerValue.TIMESTAMP).timedGet();
 
     Node node = manager.serverCache(query).getNode();
-    assertTrue(Math.abs(System.currentTimeMillis() - (Long) node.getValue()) < 1000);
```

Comment:
note: here I changed the maximum drift to 2s to match the rest of the codebase and reduce flakiness